### PR TITLE
Free fio address reg link. Use payment flow for fio address reg.

### PR DESCRIPTION
**Pull request by andreyvEze**
Снимок экрана 2020-08-12 в 17 14 58

### CODE REVIEW

File: src/components/scenes/FioAddressRegisterScene.js [hunk 1]
```diff
@@ -122,6 +137,24 @@ export class FioAddressRegisterScene extends React.Component<Props, State> {
...
 
+  registerFreeAddress = () => {
+    const { fioPlugin, fioWallets } = this.props
+    const { selectedWallet } = this.state
+    if (!fioPlugin) return
+    if (!fioWallets.length) return
+    if (!selectedWallet) return
+    const publicKey = selectedWallet.publicWalletInfo.keys.publicKey
+    const url = `https://reg.fioprotocol.io/address/edgefree?publicKey=${publicKey}`
+    Linking.canOpenURL(url).then(supported => {
```
Comment (andreyvEze) [inline src/components/scenes/FioAddressRegisterScene.js:148]: SafariView
Put in utils functions

File: src/components/scenes/FioAddressRegisterScene.js [hunk 2]
```diff
@@ -480,6 +522,11 @@ const rawStyles = {
     paddingLeft: THEME.rem(0.75),
     paddingRight: THEME.rem(0.75),
     paddingVertical: THEME.rem(0.75)
+  },
+  link: {
+    padding: scale(2),
```
Comment (andreyvEze) [inline src/components/scenes/FioAddressRegisterScene.js:527]: THEME.rem

File: src/util/utils.js [hunk 3]
```diff
@@ -600,3 +602,27 @@ export async function asyncWaterfall(asyncFuncs: Array<AsyncFunction>, timeoutMs
...
+        Linking.canOpenURL(url).then(supported => {
+          if (supported) {
+            Linking.openURL(url)
+          } else {
+            throw new Error(`Don't know how to open URI: ${url}`)
+          }
+        })
+      })
+  }
+  Linking.canOpenURL(url).then(supported => {
```
Comment (andreyvEze) [inline src/util/utils.js:621]: create one more method for this

**Comment by andreyvEze** [comment]
approved

**Review by paullinator — APPROVED** [review]
Approved by @thehobbit85

**Pull request merged**